tests/integration_tests/http_api/util/test_case.py

```diff
@@ -136,6 +136,27 @@ def create_changefeed(sink_uri):
     resp = rq.post(url, data=data, headers=headers)
     assert "ErrDispatcherFailed" in resp.text, f"{resp.text}"
 
+    # create changefeed fail because glue schema config is invalid
+    url = BASE_URL1_V2+"/changefeeds"
```

Comment:
To be compatible with next-gen mode, please add the parameter `keyspace=keyspace1` to the URL.